docker/worker-tulip/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3-slim
```

Comment:
Is there a particular reason for not using the pypy image?

Reply:
Yes - the ability to `pip install tulip-python`